web-frame/src/app-shared/config/defaults/available-feature-modules.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license
+ * Copyright (c) 2020 THREEANGLE SOFTWARE SOLUTIONS SRL
+ * Available under MIT license webFrame/LICENSE
+ */
+import { LoadChildrenCallback } from '@angular/router';
+
+/* tslint:disable:typedef */
+export const AVAILABLE_FEATURE_MODULES: Record<string, LoadChildrenCallback> = {
+  account: () => import('~app-feature/account-feature').then(m => m.default),
+  profile: () => import('~app-feature/profile-feature').then(m => m.default),
+  sandbox: () => import('~app-feature/sandbox-feature').then(m => m.default),
```

Comment:
By using a single file, the sandbox module is always built, but since it's contained in a separate file, we can just ignore it.

Alternatively, we could use `fileReplacements` and copy this information into 2 different files. and remove the sandbox from the `.prod` version.